app/views/devise/confirmations/show.html.slim

```diff
@@ -10,6 +10,9 @@
       = simple_form_for(resource, as: resource_name, url: confirm_path,  html: { role: 'form', autocomplete: 'off' }) do |f|
         = f.error_notification
         = f.input :password, required: true, input_html: { autofocus: true }
+        = render 'devise/shared/password_strength'
```

Comment:
my thinking was that this module might go on other pages too (i.e., reset / change password), though maybe we should just inline until we do that?